docs/environment_variables.md

```diff
@@ -6,20 +6,21 @@ These are the environment variables set on every WAGI requests:
 
 ```bash
 # The name of the route that matched
-X_MATCHED_ROUTE="/envwasm"  # for example.com/envwasm
+X_MATCHED_ROUTE="/envwasm/..."  # for example.com/envwasm
 # The value of the HTTP Accept header from the client. This could be empty.
 HTTP_ACCEPT="*/*"
 # The HTTP method (GET/POST/PUT/etc) sent by the client
 REQUEST_METHOD="GET"
-# The protocol that the server is using. Normally this is "http" or "https"
-SERVER_PROTOCOL="http"
+# The protocol that the server is using. Usually it is HTTP/1.1
```

Comment:
Should we mention it is always HTTP/1.1 for now?
Do we want to add a note in the docs about HTTP/2?

Reply:
I think I am going to leave it just b/c I know that Wagi.net points to these docs in some places. And IIRC, it can do HTTP/2